build/PackageDiffIgnore.xml

```diff
@@ -444,6 +444,48 @@
 				<Member 
 					fullName="System.Void Windows.UI.Xaml.Controls.Primitives.RangeBaseValueChangedEventArgs..ctor()"
 					reason="Not part of the UWP API."/>
+				<Member 
+					fullName="System.Void Windows.UI.Input.ManipulationStartedEventArgs..ctor()"
+					reason="Not part of the UWP API." />
+				<Member 
+					fullName="System.Void Windows.UI.Input.ManipulationUpdatedEventArgs..ctor()"
+					reason="Not part of the UWP API." />
+				<Member 
+					fullName="System.Void Windows.UI.Input.ManipulationInertiaStartingEventArgs..ctor()"
+					reason="Not part of the UWP API." />
+				<Member 
+					fullName="System.Void Windows.UI.Input.ManipulationCompletedEventArgs..ctor()"
+					reason="Not part of the UWP API." />
+				<Member 
```

Comment:
Not sure to understand the reason for this. The get/set methods are there.

Reply:
Me neither... Sounds like a bug of the diff tool, so I just workaround it.